commerce-add-ons/payment-providers/buckaroo/README.md

```diff
@@ -0,0 +1,24 @@
+---
+description: Getting Started with the Buckaroo payment provider for Umbraco Commerce.
+---
+
+# Overview
+
+The Buckaroo payment provider adds functionality to capture payments via the [Buckaroo](https://buckaroo.com) payment gateway. It is a fully featured payment provider allowing full control of the payment flow directly from the Umbraco Commerce backoffice.
+
+## [Install Buckaroo](../install-payment-providers.md)
+
+## In this Section
+
+In this section, we will guide you through the key steps necessary to get you started with the Buckaroo payment provider for Umbraco Commerce.
+
+It is assumed that before we begin that you already have an Umbraco website configured and Umbraco Commerce installed. If you are not at this stage yet, please read the [core Umbraco Commerce documentation](https://docs.umbraco.com/umbraco-commerce/) to learn how to get started.
+
+## Useful links
+
+Below are a few useful links that might come in handy should you wish to learn more about the provider itself, and the Buckaroo API.
+
+* [Payment provider source code](https://github.com/umbraco/Umbraco.Commerce.PaymentProviders.Buckaroo)
+* [Payment provider issue tracker](https://github.com/umbraco/Umbraco.Commerce.PaymentProviders.Buckaroo/issues)
```

Comment:
those links do not work. @mattbrailsford will these links be available in the future?

Reply:
@alina-tincas  yes, they will be available in the future. The repository is private right now.

Reply:
Once the repo is made public yes they will 👍